.github/workflows/gh-actions-release.yml

```diff
@@ -0,0 +1,79 @@
+name: 'GH Actions - Release'
+
+on:
+  release:
+    types:
+      - created
+      - edited
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          # only build one Deb File
+          #- FROM:     'ubuntu:focal'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'ubuntu:bionic'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'ubuntu:xenial'
+          #  COMPILER: 'gcc'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          #- FROM:     'debian:buster'
+          #  COMPILER: 'clang'
+          #  FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'debian:stretch'
+            COMPILER: 'gcc'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'opensuse/leap'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:33'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'fedora:32'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+          - FROM:     'centos:8'
+            COMPILER: 'clang'
+            FLAGS:    '-DUSE_PYTHON_3=ON'
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        # We must fetch at least the immediate parents so that if this is
+        # a pull request then we can checkout the head.
+        fetch-depth: 2
+        submodules: false
+
+    # where in a pull request we want to move to the head of the PR and not be at the
+    # merge commit; here we want to be at the HEAD of the ref
+
+    - name: Extract tag name etc
+      shell: bash
+      run: |
+        echo "TAG_NAME=$(echo ${GITHUB_REF#refs/tags/} | sed 's/\//_/g')" >> $GITHUB_ENV
+    - name: Test tag name etc
+      run: |
+        echo "${TAG_NAME}"
+    - name: Run CI
+      env:
+        FROM:       ${{ matrix.FROM }}
+        COMPILER:   ${{ matrix.COMPILER }}
+        FLAGS:      ${{ matrix.FLAGS }}
+        MY_OS_NAME: linux
+        IS_RELEASE: 1
+      run: script/cibuild $FLAGS
+    - name: Upload the artifacts
+      uses: skx/github-action-publish-binaries@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        args: 'packages/*'
```

Comment:
We should probably leave out the source packages, in similar fashion to https://github.com/vegastrike/Vega-Strike-Engine-Source/pull/402 , so that we don't end up with file conflicts during the release process.

Reply:
I'll have to add that to Master as well.